Source/td.cpp

```diff
@@ -1,11 +1,13 @@
 #include <iostream>
 #include <SFML/Graphics.hpp>
-#include "screens.hpp"
+#include "Screens/Screens.hpp"
 
 int main()
 {
 	//Applications variables
-	std::vector<cScreen*> Screens;
+	std::vector<ScreenBase*> Screens;
+
+	// Сомнительное резервирование памяти (зочем?)
```

Comment:
Yes, I understand, there is no any performance improvment in this case. I hope, it will be interesting for u
![image](https://user-images.githubusercontent.com/21985069/43849139-61249e6e-9b3d-11e8-89e5-e952fcf4fd34.png)
![image](https://user-images.githubusercontent.com/21985069/43849181-7cee582e-9b3d-11e8-841e-a2d4e6f35fa8.png)